src/cpp/rtps/transport/test_UDPv4Transport.cpp

```diff
@@ -407,13 +413,20 @@ bool test_UDPv4Transport::packet_should_drop(
 }
 
 bool test_UDPv4Transport::log_drop(
-        const octet* buffer,
-        uint32_t size)
+        const std::array<asio::const_buffer, max_required_buffers>& send_buffers,
+        uint32_t total_bytes)
```

Comment:
There is a `-Wunused-parameter` warning: unused parameter `total_bytes`.

Reply:
Unused parameter removed in a551f3a0c7d0e83d05f6cd6d5d18b8e774cd69c9